examples/windows-audit/habitat/default.toml

```diff
@@ -0,0 +1,17 @@
+# You must accept the Chef License to use this software: https://www.chef.io/end-user-license-agreement/
+# Change [chef_license] from acceptance = "undefined" to acceptance = "accept-no-persist" if you agree to the license.
+
+interval = 1800
+splay = 1800
+splay_first_run = 0
+log_level = 'warn'
+report_to_stdout = true
+
+[chef_license]
+acceptance = "undefined"
+
+[automate]
+enable = true
+server_url = "https://<automate_url>/data-collector/v0/"
+token = '<automate_token>'
+user = '<automate_user>'
```

Comment:
This file needs a newline at the end. I'm surprised the lint didn't catch this? I'll have to fix that.

Please also set `enable = false` by default.

Reply:
I'll fix the newline, however, as mentioned in the other comment, I'm confused as to why we'd want this disabled by default.

Reply:
We have the Automate reporting disabled as default as not everyone has Automate stood up. Currently, even the InSpec run is reported to the log so in the cause of windows `c:\hab\svc\windows-service\logs\Habitat.log`